docs/docs/policy-performance.md

```diff
@@ -977,6 +977,66 @@ This feature can be enabled for `opa run`, `opa eval`, and `opa bench` by settin
 
 Users are recommended to do performance testing to determine the optimal configuration for their use case.
 
+## Performance Metrics
```

Comment:
We still seem to have the per built in metrics here in this doc as well as in the built in docs themselves, I think they're better in the built in docs only.